web/themes/custom/pineapple/components/_webform.twig

```diff
@@ -23,6 +23,7 @@
 #}
 {% include "@components/_heading.twig" with {
   heading_text: title,
+  heading_attributes: create_attribute().addClass('subheading'),
```

Comment:
What determines something is a subheading?

Reply:
This class is added to all headings which are supposed to look like h2